src/components/button/templates/component/style.js

```diff
@@ -217,6 +240,14 @@ export let componentStyle = `
         .paypal-button .paypal-button-content .logo.logo-paypal {
             height: 24px;
         }
+        
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-paypal {
+            height: 19px;
+        }
+        
+        .paypal-button .paypal-button-content .logo.logo-venmo {
+            height: 13px;
```

Comment:
Just thinking... why wouldn't the venmo logo be the same height as the paypal logo? wouldn't that make things way simpler?